src/CoreWCF.ConfigurationManager/src/CoreWCF/Configuration/ConfigurationManagerExtensions.cs

```diff
@@ -4,13 +4,17 @@
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.DependencyInjection.Extensions;
 using Microsoft.Extensions.Options;
+using System.IO;
+using System;
 
 namespace CoreWCF.Configuration
 {
     public static class ConfigurationManagerExtensions
     {
         public static IServiceCollection AddServiceModelConfigurationManagerFile(this IServiceCollection builder, string path)
         {           
+            if (!File.Exists(path)) { throw new ArgumentException($"File {path} Does not Exist"); }
```

Comment:
Can you add this to the string resources file (Strings.resx). You will need to switch to the old string format style using `{0}` and use `SR.Format(SR.ResourceName, path)`. Also the casing needs to be changed to follow regular English grammar, so the words does and exist should have a lowercase first letter.